reconcile/sendgrid_teammates.py

```diff
@@ -47,28 +55,52 @@ def fetch_current_state(sg_client):
             # we want to ignore the root account (owner account)
             continue
 
-        t = Teammate(teammate['email'], pending=False)
+        t = Teammate(teammate['email'])
         state.append(t)
 
     return state
 
 
+def raise_if_error(response):
+    """
+    Raises an SendGridAPIError if the request has returned an error
+    """
+    if int(response.status_code / 100) != 2:
```

Comment:
`if response.status_code >= 300:` is 3x faster :)

Reply:
good idea, yes :D